### PR TITLE
Cloudsplaining Testing Start

### PR DESCRIPTION
**Pull request by njgibbon**
Hello,

Start for organising testing for Cloudsplaining and CloudFormation checks.

Related to issue: https://github.com/bridgecrewio/checkov/issues/1037

This moves managed policy testing and covers the checks related to the above issue which will now pass and ignore 'Deny' Actions given 0.4.1 > Cloudsplaining usage.

**License**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ tqdm = "*"
 update_checker = "*"
 semantic_version = "*"
 packaging = "*"
-cloudsplaining = ">=0.3.2"
+cloudsplaining = ">=0.4.1"
```
Comment (schosterbarak) [inline Pipfile:37]: thanks @njgibbon . Can you please updadte setup.py file too?

Comment (njgibbon) [reply Pipfile:37]: Done that now. Also moved the test file structure around so it's not nested after some cf-lint feedback on one of the previous commits.

**Comment by schosterbarak** [comment]
Great work. should i close https://github.com/bridgecrewio/checkov/issues/1037 after merging?

**Comment by njgibbon** [comment]
Yes I think this should cover all of 1037 as only these 2 checkov checks were affected by the cloudsplaining change. I'll be adding more test coverage and some more IAM checks for CF in the next week or so 🙂

File: Pipfile (hunk 1 above)
Comment (schosterbarak) [reply Pipfile:37]: I think we need too update pipfile.lock to

Comment (njgibbon) [reply Pipfile:37]: Updating that file now

**Review by nimrodkor — APPROVED** [review]
GJ!

**Pull request merged**